cli/src/native.rs

```diff
@@ -84,6 +88,37 @@ pub(crate) async fn run(args: Params) -> Result<()> {
     }
 }
 
+async fn open_db(path: Option<PathBuf>, network_id: &str) -> Result<Db> {
+    if let Some(path) = path {
+        let db = spawn_blocking(|| sled::open(path)).await??;
+        return Ok(db);
+    }
+
+    let cache_dir =
+        ProjectDirs::from("co", "eiger", "lumina").context("Couldn't find lumina's cache dir")?;
+    let mut cache_dir = cache_dir.cache_dir().to_owned();
+
+    // TODO: remove it after we publish a new version to crates.io?
+    // If we find an old ('celestia') cache dir, move it to the new one.
+    if let Some(old_cache_dir) = ProjectDirs::from("co", "eiger", "celestia") {
+        let old_cache_dir = old_cache_dir.cache_dir();
+        if old_cache_dir.exists() && !cache_dir.exists() {
+            warn!(
+                "Migrating old cache dir to a new location: {} -> {}",
+                old_cache_dir.display(),
+                cache_dir.display()
+            );
+            fs::rename(old_cache_dir, &cache_dir).await?;
+        }
+    }
+
+    cache_dir.push(network_id);
+    // TODO: should we create there also a subdirectory for the 'db'
+    // in case we want to put there some other stuff too?
+    let db = spawn_blocking(|| sled::open(cache_dir)).await??;
+    Ok(db)
+}
```

Comment:
Should we provide this as a helper in `lumina-node`? Before this PR a user could use `SledStore` in `Node` and continue from the state `lumina-cli` had. We can name it `open_default_sled_db`.

Reply:
I actually liked the removal of this logic from the `lumina-node` but I can move it back there, wdyt @fl0rek

Reply:
For me the main issue is that users need to inspect our code to figure out that we are using cache directory of `ProjectDirs` and then do the opening of the sled db. If we provide them a helper, we are making this part of our API instead.

Reply:
Maybe we can add `pub fn lumina_default_db() -> PathBuf` for that case?